src/components/Products/ProductsFilter.js

```diff
@@ -0,0 +1,345 @@
+
+                <Text id="filter-priceval" className="txt-md txt-primary">{filterState.filterPrice}</Text>
+
+            </div>
+
+
+            <div className='mg-btm-lg'>
+
+                <Text classes='txt-md txt-cap mg-btm-xs'>
+                    sort by price
+                </Text>
+
+                <div className='flx flx-min-center'>
+
+                    <input type='radio' name='price-grp' id='price-asc'
+                        onChange={(e) => handlePriceSortOrderChange(e)}
+                        value='asc' checked={isSortInputIncluded('asc', filterState.priceSortOrder)} className='mg-right-xs' />
+
+                    <label htmlFor='price-asc' className='txt-cap'>low to high</label>
+
+                </div>
+
+                <div className='flx flx-min-center'>
+
+                    <input type='radio' name='price-grp' id='price-desc' onChange={(e) => handlePriceSortOrderChange(e)} value='dsc' checked={isSortInputIncluded('dsc', filterState.priceSortOrder)} className='mg-right-xs' />
+
+                    <label htmlFor='price-desc' className='txt-cap'>high to low</label>
+
+                </div>
+
+            </div>
+
+
+            <div className='mg-btm-lg flx flx-column'>
+
+                <Text classes='txt-md txt-cap mg-btm-xs'>
+                    gender
+                </Text>
+
+                <div className='flx flx-min-center'>
+
+                    <input type='checkbox' id='gender-m' value='male' checked={isInputIncluded('male', filterState.genderChecks)} onChange={(e) => handleGenderChecks(e)} className='mg-right-xs' />
+
+                    <label htmlFor='gender-m' className='txt-cap'>men</label>
+
+                </div>
+
+                <div className='flx flx-min-center'>
+
+                    <input type='checkbox' id='gender-f' value='female' checked={isInputIncluded('female', filterState.genderChecks)} onChange={(e) => handleGenderChecks(e)} className='mg-right-xs' />
+
+                    <label htmlFor='gender-f' className='txt-cap'>women</label>
+
+                </div>
+
+            </div>
+
+
+            <div className='mg-btm-lg'>
+
+                <Text classes='txt-md txt-cap mg-btm-xs'>
+                    brands
+                </Text>
+
+                <div className='flx flx-min-center'>
+
+                    <input type='checkbox' id='nike-category' value='nike' checked={isInputIncluded('nike', filterState.brandChecks)} onChange={(e) => handleBrandCheck(e)} className='mg-right-xs' />
+
+                    <label htmlFor='nike-category' className='txt-cap'>nike</label>
+
+                </div>
+
+                <div className='flx flx-min-center'>
+
+                    <input type='checkbox' id='adidas-category' value='adidas' checked={isInputIncluded('adidas', filterState.brandChecks)} onChange={(e) => handleBrandCheck(e)} className='mg-right-xs' />
+
+                    <label htmlFor='adidas-category' className='txt-cap'>adidas</label>
+
```

Comment:
Try to take this in object and then map, then its dynamic

Reply:
thanks @ShekharDhangar. i will add the logic later. this is only for ui.